src/components/views/Game.js

```diff
@@ -68,6 +68,8 @@ export default function Game() {
     const [roundsPlayed, setRoundsPlayed] = useState("");
     const [team1Size, setTeam1Size] = useState(0);
     const [team2Size, setTeam2Size] = useState(0);
+    const [isBuzzerPressed, setIsBuzzerPressed] = useState(false);
```

Comment:
I would rename the variable to something like "buzzerWasPressed" or "buzzerWasPressedOnce".